api_tests/applications/views/test_application_detail.py

```diff
@@ -1,33 +1,33 @@
+import pytest
 import mock
 
-from nose.tools import *  # flake8: noqa
-
 from website.models import ApiOAuth2Application, User
 from website.util import api_v2_url
-
 from tests.base import ApiTestCase
+from api_tests.utils import assert_dict_contains_subset
+from tests.json_api_test_app import JSONAPITestApp
 from osf_tests.factories import ApiOAuth2ApplicationFactory, AuthUserFactory
 
 
 def _get_application_detail_route(app):
-    path = "applications/{}/".format(app.client_id)
+    path = 'applications/{}/'.format(app.client_id)
     return api_v2_url(path, base_route='/')
 
 
 def _get_application_list_url():
-    path = "applications/"
+    path = 'applications/'
     return api_v2_url(path, base_route='/')
 
+@pytest.mark.django_db
+class TestApplicationDetail:
 
-class TestApplicationDetail(ApiTestCase):
+    @pytest.fixture(autouse=True)
     def setUp(self):
-        super(TestApplicationDetail, self).setUp()
-
-        self.user1 = AuthUserFactory()
-        self.user2 = AuthUserFactory()
-
-        self.user1_app = ApiOAuth2ApplicationFactory(owner=self.user1)
-        self.user1_app_url = _get_application_detail_route(self.user1_app)
+        self.app = JSONAPITestApp()
+        self.user_one = AuthUserFactory()
+        self.user_two = AuthUserFactory()
+        self.user_one_app = ApiOAuth2ApplicationFactory(owner=self.user_one)
+        self.user_one_app_url = _get_application_detail_route(self.user_one_app)
```

Comment:
Drive-by comment: We should be moving away from the xunit style (setUp/tearDown) of tests and use pytest fixtures instead. This will allow individual tests to instantiate the minimum subset of fixtures that are needed.